### PR TITLE
Implement Pipeds stat cleaner

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:

Ensure remove staled Pipeds' stat using cleaner.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/ops/staledpipedstatcleaner/cleaner.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		select {
+		case <-ctx.Done():
+			s.logger.Info("staledPipedStatCleaner has been stopped")
+			return nil
+
+		case <-t.C:
+			start := time.Now()
+			if err := s.flushStaledPipedStat(); err == nil {
+				s.logger.Info("successfully cleaned staled pipeds stat", zap.Duration("duration", time.Since(start)))
+			}
```
Comment (nakabonne) [inline pkg/app/ops/staledpipedstatcleaner/cleaner.go:61]: doesn't handle the error?

Comment (nakabonne) [reply pkg/app/ops/staledpipedstatcleaner/cleaner.go:61]: I'd say stop emitting log within flushStaledPipedStat, instead, wrap them using fmt.Errorf like `fmt.Errorf("ailed to fetch piped stats from cache", err)`. Then emit them as a log here

File: pkg/app/ops/staledpipedstatcleaner/cleaner.go [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	"fmt"
+	"time"
+
+	"go.uber.org/zap"
+
+	"github.com/pipe-cd/pipe/pkg/cache"
+	"github.com/pipe-cd/pipe/pkg/model"
+)
+
+var (
```
Comment (nakabonne) [inline pkg/app/ops/staledpipedstatcleaner/cleaner.go:30]: should be const

File: pkg/app/ops/staledpipedstatcleaner/cleaner.go (hunk 1 above)
Comment (khanhtc1202) [reply pkg/app/ops/staledpipedstatcleaner/cleaner.go:61]: Okay, I hear you 👍 Just wanna keep this cleaner the same as other cleaners but I like your suggestion. Will adopt that 👌

**Comment by nakabonne** [comment]
Nice one
/approve

File: pkg/app/ops/staledpipedstatcleaner/cleaner.go (hunk 1 above)
Comment (khanhtc1202) [reply pkg/app/ops/staledpipedstatcleaner/cleaner.go:62]: @nakabonne sorry, I mean I will adopt your suggestion in this PR 🙏 
Addressed by https://github.com/pipe-cd/pipe/pull/2287/commits/c92b4db970abc14f761ce8ad313be0ea42da357a 🙏 
PTAL 👀

Comment (nakabonne) [reply pkg/app/ops/staledpipedstatcleaner/cleaner.go:62]: i got wrong

**Pull request merged**